airflow/utils/email.py

```diff
@@ -87,8 +91,10 @@ def send_email_smtp(
     """
     smtp_mail_from = conf.get('smtp', 'SMTP_MAIL_FROM')
 
+    mail_from = smtp_mail_from or from_email
```

Comment:
Should we automatically read the `from_email` configuration here as well?

Reply:
What do you mean automatically? It is read in `send_email` and then passed to this method

Reply:
If a user calls this function directly (not via `send_email`), I feel they should still automatically get `FROM_EMAIL` if `SMTP_MAIL_FROM` is not set, instead of needing to pass it in manually.

(This also reminds me, should we change the former to `EMAIL_FROM` instead for consistency? Or maybe even `DEFAULT_FROM` or just `FROM`?)

Reply:
If this is implemented in `send_email_smtp` then the same convention should be followed on other email backends as well. Wouldn't this cause too much repetition?

Reply:
Yes, but I don’t think it’s a bad thing since those functions are fundamentally different. It also allows a new backend to choose to *not* respect the `email_from` config if that makes sense.